packages/aws-cdk-lib/aws-synthetics/lib/canary.ts

```diff
@@ -229,6 +230,51 @@ export interface CanaryProps {
    * @default - no rules applied to the generated bucket.
    */
   readonly artifactsBucketLifecycleRules?: Array<s3.LifecycleRule>;
+
+  /**
+   * Canary Artifacts in S3 encryption configuration.
+   * Artifact encryption is only supported for Node.js runtime.
```

Comment:
```suggestion
   *
   * Artifact encryption is only supported for canaries that use Synthetics runtime
   * version `syn-nodejs-puppeteer-3.3` or later.
```
It seems more appropriate to keep it consistent with CloudFormation [docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-synthetics-canary-artifactconfig.html).